LocalSettings.php

```diff
@@ -135,6 +135,31 @@
 wfLoadSkin( 'MinervaNeue' );
 wfLoadSkin( 'Vector' );
 
+
+# Enabled extensions. Most of the extensions are enabled by adding
+# wfLoadExtension( 'ExtensionName' );
+# to LocalSettings.php. Check specific extension documentation for more details.
+# The following extensions were automatically enabled:
+wfLoadExtension( 'BetaFeatures' );
+wfLoadExtension( 'CentralNotice' );
+wfLoadExtension( 'Cite' );
+wfLoadExtension( 'Echo' );
+wfLoadExtension( 'EventBus' );
+wfLoadExtension( 'EventLogging' );
+wfLoadExtension( 'EventStreamConfig' );
+wfLoadExtension( 'Gadgets' );
+wfLoadExtension( 'GlobalPreferences' );
+wfLoadExtension( 'MobileFrontend' );
+wfLoadExtension( 'Popups' );
+wfLoadExtension( 'RelatedArticles' );
+wfLoadExtension( 'SandboxLink' );
+wfLoadExtension( 'Scribunto' );
+wfLoadExtension( 'TemplateStyles' );
```

Comment:
I don't think we need Scribunto or TemplateStyles since we have no Module or Template pages on the wiki.

Since this is UI testing, EventLogging, WikimediaEvents, EventBhs and EventStreamConfig also not likely needed.

Reply:
Good point, I'll remove!